ExternalTasks/NodeJs/main.js

```diff
@@ -13,7 +13,7 @@ const identity = {
     token: 'ZHVtbXlfdG9rZW4=',
 };
 
-const TOPIC = 'Etikett-ausdrucken';
+const TOPIC = 'AktivierungsemailSenden';
```

Comment:
Wie oben, in Typescript, bzw. Node schreiben auf auch Konstanten immer so:

```suggestion
const topic = 'AktivierungsemailSenden';
```